fsw/cfe-core/src/inc/cfe_sb.h

```diff
@@ -562,7 +560,7 @@ int32 CFE_SB_UnsubscribeLocal(CFE_SB_MsgId_t MsgId, CFE_SB_PipeId_t PipeId);
 **
 ** \param[in]  MsgPtr       A pointer to the message to be sent.  This must point
 **                          to the first byte of the software bus message header
-**                          (#CFE_SB_Msg_t). 
+**                          ("::"CFE_SB_Msg_t). 
```

Comment:
Curious why old reference didn't work?  What does `"::"` do?

Reply:
Part of the autolink system: http://www.doxygen.nl/manual/autolink.html (item 3 under " Links to functions"

Reply:
I double checked, I'm noticing that the literal "::" appears as well. I missed that, I may need to revisit

Reply:
Did the `#` cause an error/warning?  Note 3 from that reference seems to say `#` should work?  "For Javadoc compatibility a # may be used instead of a :: in the patterns above."

Reply:
It was, yes, which is why I changed it, and got no error as a result.